.pre-commit-config.yaml

```diff
@@ -644,12 +642,6 @@ repos:
         files: ^airflow/www/static/js/
         entry: scripts/ci/static_checks/www_lint.sh
         pass_filenames: false
-      - id: bats-in-container-tests
-        name: Run in container bats tests
-        language: system
-        entry: ./scripts/ci/pre_commit/pre_commit_in_container_bats_test.sh
-        files: ^tests/bats/in_container/.*\.bats$|^scripts/in_container/.*sh
-        pass_filenames: false
```

Comment:
Seems unrelated to pylint?

Reply:
Oh it moved.

Reply:
It's moved to after the `build` step - the check failed if the build has not been run before (discovered it while testing this change). I can separate it out, but since we are changing the pre-commit anyway quite heavily, fixing it in the "big" pylint cleanup PR might be a good idea.

Reply:
Nah, it's good.